_opensearch/rest-api/cluster-health.md

```diff
@@ -38,14 +38,20 @@ Parameter | Type | Description
 :--- | :--- | :---
 expand_wildcards | Enum | Expands wildcard expressions to concrete indexes. Combine multiple values with commas. Supported values are `all`, `open`, `closed`, `hidden`, and `none`. Default is `open`.
 level | Enum | The level of detail for returned health information. Supported values are `cluster`, `indices`, and `shards`. Default is `cluster`.
+<<<<<<< improve_cluster_health
+local | Boolean | Whether to return information from the local node only instead of from the master node. Default is false.
+cluster_manager_timeout<br>(deprecated `master_timeout`) | Time | The amount of time to wait for a connection to the cluster manager node. Default is 30 seconds.
+=======
 local | Boolean | Whether to return information from the local node only instead of from the cluster manager node. Default is false.
 cluster_manager_timeout | Time | The amount of time to wait for a connection to the cluster manager node. Default is 30 seconds.
+>>>>>>> main
 timeout | Time | The amount of time to wait for a response. If the timeout expires, the request fails. Default is 30 seconds.
 wait_for_active_shards | String | Wait until the specified number of shards is active before returning a response. `all` for all shards. Default is `0`.
+wait_for_nodes | String | Wait for N number of nodes. Use `12` for exact match, `>12` and `<12` for range.
 wait_for_events | Enum | Wait until all currently queued events with the given priority are processed. Supported values are `immediate`, `urgent`, `high`, `normal`, `low`, and `languid`.
 wait_for_no_relocating_shards | Boolean | Whether to wait until there are no relocating shards in the cluster. Default is false.
 wait_for_no_initializing_shards | Boolean | Whether to wait until there are no initializing shards in the cluster. Default is false.
-wait_for_status | Enum | Wait until the cluster is in a specific state or better. Supported values are `green`, `yellow`, and `red`.
+wait_for_status | Enum | Wait until the cluster health reaches specific status or better. Supported values are `green`, `yellow`, and `red`.
```

Comment:
I do like the word `status` better here. However, instead of "specific", should we try something like, "Wait until the cluster health reaches the specified statues or better?"